composer.json

```diff
@@ -14,7 +14,10 @@
         }
     },
     "require": {
-        "php": "^7.1"
+        "php": "^7.1",
+        "aerospike/aerospike-client-php": "~7.0",
+        "doctrine/cache": "^1.8",
```

Comment:
This should be imho only suggested. You could use pure symfony/cache PSR-6/PSR-16 adapter, without even having doctrine/cache installed.

Reply:
Marked as suggested